build/compile_descriptor

```diff
@@ -20,20 +20,26 @@ is_up_to_date () {
 
 set -e
 
+if [ -d "./ebin" ]; then
+    OUT="./ebin"
+else
+    OUT="$REBAR_DEPS_DIR/gpb/ebin"
+fi
+
```

Comment:
```
% make clean; rebar compile
```
works
but
```
% make clean; rebar3 compile
rm -f include/gpb_version.hrl ...
...
===> Verifying dependencies...
===> Compiling gpb
Compiling descriptor.proto...
{"init terminating in do_boot",{undef,[{gpb_compile,c,[['descriptor.proto']],[]},{init,start_it,1,[]},{init,start_em,1,[]}]}}

Crash dump is being written to: erl_crash.dump...done
init terminating in do_boot ()
===> Hook for compile failed!

```
(my gpb is at dde28cb at the moment, rebar3 is `rebar 3.2.0+build.3505.ref21571ad on Erlang/OTP 18 Erts 7.3`)